io-engine/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -197,6 +206,8 @@ pub enum NexusStatus {
     Degraded,
     /// Online
     Online,
+    /// Shutdown
+    Shutdown,
```

Comment:
Shutdown is very misleading IMHO. What is the difference between unshared and shutdown for example. What is the use case for such a state?

Reply:
Such a state is used in nexus switchover: we keep nexus published but pause its I/O subsystem (which keeps the connected initiator alive while we're recreating the nexus on another node).

Reply:
So why the new state then? IIRC there already is a paused state for the nexus?

Reply:
Shutdown operation performs a bit more than just pauses nexus (like closing all its replicas and cancelling all rebuild jobs). This new state is to denote that nexus underwent the former.